minecraft/plugin-shading.md

```diff
@@ -0,0 +1,11 @@
+---
+tag: plugin shading
+alias: ["gradle shading", "minecraft shading"]
+---
+
+Nehmen wir an unsere Library befindet sich nicht im Maven Central und wir können den Library Loader nicht verwenden.
```

Comment:
```suggestion
Nehmen wir an unsere Library befindet sich nicht im Maven Central und wir können somit den Library Loader nicht verwenden.
```

Reply:
Switched to and/or to also take users into account on version which do not have the library loader